src/python/libcgroup.pyx

```diff
@@ -627,6 +627,40 @@ cdef class Cgroup:
 
         return pid_list
 
+    def get_current_controller_path(self, pid, controller):
```

Comment:
Also, it doesn't appear that we're using the Cgroup instance at all.  Should this method be made a `@staticmethod`?

Reply:
agreed, will make it staticmethod